### PR TITLE
 Read the VERSION file from the serviceAccessConfig lib

### PR DESCRIPTION
**Pull request by bear454**
... wherever that may be.

### CODE REVIEW

**Comment by bear454** [comment]
Good grief.

File: usr/sbin/serviceAccessConfig [hunk 1]
```diff
@@ -40,13 +40,15 @@ Options:
 
 import configparser
 import logging
+import os
+import serviceAccessConfig.generatorfactory as factory
```
Comment (rjschwei) [inline usr/sbin/serviceAccessConfig]: please move the custom imports below the Python module imports, i.e.

```
import sys

import serviceAccessConfig.generatorfactory as factory
```

**Review by rjschwei — CHANGES_REQUESTED** [review]

File: usr/sbin/serviceAccessConfig (hunk 1 above)
Comment (bear454) [reply usr/sbin/serviceAccessConfig]: Fixed in ac15be0.

**Review by rjschwei — APPROVED** [review]

**Review by smarlowucf — APPROVED** [review]
Good. Could consider adding bumpversion to the workflow and removing version file parsing in setup.py and serviceAccessConfig.

**Pull request merged**